core/GhcMod/CabalHelper.hs

```diff
@@ -89,28 +89,17 @@ getComponents = chCached $ \distdir -> Cached {
     cacheLens = Just (lGmcComponents . lGmCaches),
     cacheFile = cabalHelperCacheFile distdir,
     cachedAction = \ _tcf (_progs, _projdir, _ver) _ma -> do
-      runCHQuery $ do
-        q <- join7
-               <$> ghcOptions
-               <*> ghcPkgOptions
-               <*> ghcSrcOptions
-               <*> ghcLangOptions
-               <*> entrypoints
-               <*> entrypoints
-               <*> sourceDirs
-        let cs = flip map q $ curry8 (GmComponent mempty)
-        return ([setupConfigPath distdir], cs)
+      cs <- runCHQuery $ components $
+             GmComponent mempty
+               CH.<$> ghcOptions
```

Comment:
The `CH.` qualifier shouldn't be required here, any particular reason you put those there?

Reply:
Probably a leftover from trying to get it to compile :) Sorry